### PR TITLE
beaker test file fixes for puppet6/dual-mode

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
I made a few changes / slight re-factoring of utilitylib.rb while testing `puppet6` branch on one of our F3-enabled n7k's. I only focused on the failed tests that you reported and got all of them working now. 
We still have to run the full suite of tests next week.  Thanks, Chris

### CODE REVIEW

**Review by Thomas-Franklin — CHANGES_REQUESTED** [review]
There seems to be a mixture of keyword arguments and positional arguments, i.e.

```
def test_set(agent, cmd, ignore_errors=false)
```

But then when calling test_set from some of the methods keyword arguments are used, i.e.

```
  test_set(agent, 'no ip domain-name test.xyz', ignore_errors: true)
```

Which means that ignore_errors will remain as `false` as this is the default in the positional argument.

Other than that then it should be fine to merge these changes as it is a nice refactor.

**Comment by chrisvanheuveln** [comment]
Thanks for the review. That suspect code actually works but it was a workaround for an ArgumentError problem we hit when passing keyword arguments through several worker methods. I figured out the proper way to do this though so I'll remove the workaround in my next commit.

fyi, here's a simple version of the original problem:

```
BAD
def com(ig: false); puts ig; end
ig=true
com(ig)
ArgumentError: wrong number of arguments (given 1, expected 0)

GOOD
def com(ig: false); puts ig; end
com(ig: ig)
=> true
```

**Comment by chrisvanheuveln** [comment]
@Thomas-Franklin That last commit should be the last of my changes for this PR. I'm getting 100% pass rate now for the set of n7k tests that were failing before my PR.
Please review when you have a chance. 
Thanks, Chris

File: tests/beaker_tests/cisco_interface/test_interface_svi.rb [hunk 1]
```diff
@@ -104,14 +104,7 @@
...
   end
-  # remove_interface(agent, intf)
-  # this command fails on fresh VMs as
-  # the interface does not exist, possibly
-  # testbed environments were not cleaned
-  # down properly, or remnants of an existing
-  # test are left over - removing the step as
-  # the cleanup in teardown should remove
-  # the interface at end of the test
+  remove_interface(agent, intf)
```
Comment (chrisvanheuveln) [inline tests/beaker_tests/cisco_interface/test_interface_svi.rb:107]: Note: `remove_interface` now ignores errors so this call should be fine.

**Comment by Thomas-Franklin** [comment]
> @Thomas-Franklin That last commit should be the last of my changes for this PR. I'm getting 100% pass rate now for the set of n7k tests that were failing before my PR.
> Please review when you have a chance.
> Thanks, Chris

I will have a look over it now and it should hopefully be merged by time you're online @chrisvanheuveln. Thanks for taking the time and resolving these issues.

**Review by Thomas-Franklin — APPROVED** [review]

**Pull request merged**